torchvision/models/shufflenetv2.py

```diff
@@ -165,16 +161,16 @@ def shufflenetv2(pretrained=False, num_classes=1000, width_mult=1, **kwargs):
 
 
 def shufflenetv2_x0_5(pretrained=False, num_classes=1000, **kwargs):
-    return shufflenetv2(pretrained, num_classes, 0.5)
+    return _shufflenetv2(pretrained, num_classes, '0.5')
```

Comment:
I think it would be nice to expose to the user paper's parameters, either by staticmethod, stand-alone function, or documentation.

Reply:
I don't know whether it would be better. @fmassa could guide you on this. But the above requested changes will make the implementation similar to that of ResNets.

Reply:
I think that I prefer the approach that is followed by `resnet` https://github.com/pytorch/vision/blob/0564df4348812604ea1d21976c81676fdaf6f9be/torchvision/models/resnet.py#L256

Paper parameters are specified in the functions (and defines the meaning of `0_5` / `1_0` etc), but the base class should be agnostic to it if possible.